testdata/scripts/node/validate/defaults-override.txtar

```diff
@@ -0,0 +1,411 @@
+# test with default override
+mkdir default_overrides
+env CL_CHAIN_DEFAULTS=default_overrides
+exec chainlink node -c config.toml -s secrets.toml validate
+cmp stdout out.txt
+
+# remove override, FinalityTagEnabled should no longer match
```

Comment:
I'm a bit confused. I see `FinalityTagEnabled` in the out.txt.

Reply:
Yes, that is the expected result of the test.